lib/libutils/ext/include/bitstring.h

```diff
@@ -144,4 +144,30 @@ typedef	unsigned char bitstr_t;
 	*(value) = _value; \
 } while (0)
 
+#define	bit_ffs_from(name, nbits, startbit, value) do { \
```

Comment:
It looks like `bit_ffs()` could then be defined as:
``` C
 #define bit_ffs(name, nbits, value) bit_ffs_from((name), (nbits), 0, (value))
```

I think it would be useful to get rid of some of the duplicated code.